Float64MultiArray q_ref
Float64MultiArray q_measured
---
Float64MultiArray q_effort